github/files/drupal7/scripts/devops/reinstall/pre_settings.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Adding database variable to settings.php
   sudo: yes
-  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"127.0.0.1\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
+  lineinfile: dest='{{ platform_settings_file }}' line='$databases = array(\"default\" => array (\"default\" => array (\"database\" => \"{{ mysql_db }}\", \"username\" => \"{{ mysql_user }}\", \"password\" => \"{{ mysql_pass }}\", \"host\" => \"{{ mysql_host }}\", \"port\" => \"\", \"driver\" => \"mysql\", \"prefix\" => \"\", ), ), );'
```

Comment:
@spheresh adding five lines is something that is strategy for adding abstraction for DB settings in terms of Drupal.
We need to have this abstraction for follovinf MVC pattern
@danylevskyi can you help @spheresh to fix this part?